diff-tests/core/driver/core_test.go

```diff
@@ -0,0 +1,462 @@
+package core
+
+import (
+	"fmt"
+	"math"
+	"testing"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/suite"
+
+	channelkeeper "github.com/cosmos/ibc-go/v3/modules/core/04-channel/keeper"
+	channeltypes "github.com/cosmos/ibc-go/v3/modules/core/04-channel/types"
+	ibctesting "github.com/cosmos/ibc-go/v3/testing"
+
+	stakingkeeper "github.com/cosmos/cosmos-sdk/x/staking/keeper"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+	appConsumer "github.com/cosmos/interchain-security/app/consumer"
+	appProvider "github.com/cosmos/interchain-security/app/provider"
+
+	simibc "github.com/cosmos/interchain-security/testutil/simibc"
+
+	slashingkeeper "github.com/cosmos/cosmos-sdk/x/slashing/keeper"
+	consumerkeeper "github.com/cosmos/interchain-security/x/ccv/consumer/keeper"
+)
+
+type CoreSuite struct {
+	suite.Suite
```

Comment:
Yeah I think it could work. I need to look into what suite actually offers more closely. I've used it so far as my original starting point was the old provider_test.go which used Suite. 

Would it be alright to merge as is and make the conversion to *testing.T an issue for future work?

Reply:
Yea seems reasonable to me. If the suite package offers value that native golang testing cannot, I'm all for it! We just wanna reduce the dependencies and complexity of testing where we can. Tracking this as an issue works 👍